src/types/JobListing.ts

```diff
@@ -0,0 +1,32 @@
+export type Job = {
+  id: number;
+  company: string;
+  logo: string;
+  new: Boolean;
+  featured: Boolean;
```

Comment:
Nitpick: To avoid confusion maybe `isFeatured` would be better

Reply:
agreed...include 2 new keys, isNew and isFeatured, in the transformJobListing